operation/fhir-operation-validate/src/main/java/org/linuxforhealth/fhir/operation/validate/ValidateOperation.java

```diff
@@ -47,38 +48,49 @@ protected OperationDefinition buildOperationDefinition() {
 
     @Override
     protected Parameters doInvoke(FHIROperationContext operationContext, Class<? extends Resource> resourceType, String logicalId,
-            String versionId, Parameters parameters, FHIRResourceHelpers resourceHelper, SearchHelper searchHelper) throws FHIROperationException {
+            String versionId, Parameters parameters, FHIRResourceHelpers resourceHelper, SearchHelper searchHelper)
+            throws FHIROperationException {
+        Objects.requireNonNull(operationContext, "operations framework prevents operationContext from being null");
+        Objects.requireNonNull(resourceType, "operations framework should prevent resourceType from being null "
+                + "because $validate must be invoked at the resource type or instance level");
```

Comment:
I moved these assertions to the top so that we can avoid all the null checks later.
The only downside to that is it makes testing a bit more complicated because we can't pass nulls.